kolibri/plugins/learn/assets/src/views/learn-page/index.vue

```diff
@@ -68,12 +122,27 @@
           params: { channel_id: this.channelId, id },
         };
       },
+      needsTrim(numOfItems) {
+        return numOfItems > carouselLimit;
+      },
+      subHeader(carouselItems, allItems) {
+        if (this.needsTrim(allItems.length)) {
+          const trArgs = {
+            subset: carouselItems.length,
```

Comment:
Could probably just make this `carouselLimit`, but thought this'd make more sense